### ISSUE TITLE
the क्वसु of षदॢँ is सेदिवस् not सेद्वस्

### ISSUE THREAD

**Issue by Stormchaser5775** [ambuda-org/vidyut#140]
https://ashtadhyayi.com/sutraani/3/2/108

**Comment by neeleshb** [comment]
गमहन.... is only for अजादि-कित् ।  क्वसु is not अजादि.  
The sequence is - द्वित्व --> .... -->  वस्वेकाजाद्घसाम् --> घसिभसोर्हलि च --> शासिवसिघसीनां च --> खरि च ।

In general note that except for सद्, वस् and श्रु, क्वसु is technically a छान्दस-प्रत्यय. However, it is seen used in kavyas in quite a few places, so we are more lenient here than लेट्. But this also means that the derivation will involve sutras that are specific to छन्दसि. For instance घसिभसोर्हलि च ।

**Comment by neeleshb** [comment]
Use 7.2.68. This sutra does have कृतद्विर्वचन-clause, so do the normal इडागम and then द्वित्व. With the इडागम, the प्रत्यय is now अजादि so गमहन.. will be invoked, which will remove अकार, which still acts स्थानिवत् for द्वित्व by द्विर्वचनेऽचि ।

**Comment by neeleshb** [comment]
No, 7.2.68 is for only 4 forms - गम् -> जग्मिवान् । हन् --> जघ्निवान् । विद् --> विविदिवान् । विश् --> विविशिवान् । 
It brings optional इडागम, so we also get जगन्वान् etc.
And दृशेश्च vartik will give ददृशिवान् / ददृश्वान् ।

**Comment by neeleshb** [comment]
Since 6.4.100 is परसूत्र so we select it.  Practically 6.4.98 is fine too. Kashika indeed seems to be using 6.4.98.  


In fact as per Kashika,  the "कृतद्विर्वचन" part is only for "एकाच्" and "आत्" components and not to घस्-component. Therefore for घस्, we do the normal process, viz the इडागम-before-द्वित्व. Tht works. But it might complicate your logic of 7.2.67. Easier would be to outrightly do द्वित्व first.

**Comment by neeleshb** [comment]
So the correct way to think about this is as follows  -
The normal sequence is "idaagama --> idaagama-dependent-karyaa -->
dvitva --> dvitva-dependent-larya".
And in the "special case", the dvitva shifts its place (along with its
karyas) giving the new sequence - "dvitva --> dvitva-dependent-karya
--> daagama --> idaagama-dependent-karya".

Now what is an idaagama-dependent-karya? It is any karya that either
happens or is blocked due to idaagama.  So it will include घुमास्था... as
well.

The central idea over here is that the first letter of a pratyay should
only be determined after all modifications are done to the pratyay. Since
इडागम is a modification on the pratyay, the decision about इडागम  (whether
to do it or not) must happen before any first-letter-based-transformation
may be done on the pratyay.
